src/apps/dfx/dfx.module.ts

```diff
@@ -0,0 +1,26 @@
+import { Module } from '@nestjs/common';
+
+import { AbstractDynamicApp } from '~app/app.dynamic-module';
+
+import { DfxContractFactory } from './contracts';
+import { DfxAppDefinition } from './dfx.definition';
+import { EthereumDfxBalanceFetcher } from './ethereum/dfx.balance-fetcher';
+import { EthereumDfxCurveTokenFetcher } from './ethereum/dfx.curve.token-fetcher';
+import { EthereumDfxStakingContractPositionFetcher } from './ethereum/dfx.staking.contract-position-fetcher';
+import { PolygonDfxBalanceFetcher } from './polygon/dfx.balance-fetcher';
+import { PolygonDfxCurveTokenFetcher } from './polygon/dfx.curve.token-fetcher';
+import { PolygonDfxStakingContractPositionFetcher } from './polygon/dfx.staking.contract-position-fetcher';
+
+@Module({
```

Comment:
Please use `Register.AppModule` as in the other app modules (this was a recent change on our side)

Reply:
Fix https://github.com/Zapper-fi/studio/pull/347/commits/4ce53b46e746eba68bd9ac2e1ed07e3d11c02efc